multiuser_service_windows_test.go

```diff
@@ -78,10 +77,12 @@ func TestConfigureAndRemoveService(t *testing.T) {
 	}
 	elog.Close()
 
-	// this is unrealistic, usually the service is marked for deletion
+	// usually the service is marked for deletion
 	// but not actually removed until reboot
 
-	// TODO
+	cleanupService(t, name)
```

Comment:
What made you decide to move this out of the defer block?

Reply:
The commented out blocks below, if uncommented, would happen before a deferred call to `cleanupService()`